content/doc/book/pipeline/getting-started.adoc

```diff
@@ -7,9 +7,11 @@ title: Getting Started with Pipeline
 :email: jenkinsci-docs@googlegroups.com
 :sectanchors:
 :toc:
-:imagesdir: /doc/book/resources
+ifdef::env-github[:imagesdir: ../resources]
+ifndef::env-github[:imagesdir: ../../resources]
```

Comment:
@gilesgas @MarkEWaite 
Whoa!  Try using the view button on GitHub with this change applied.  We can actually see images as well.  
This makes the case for a style guideline requiring relative linking much stronger.

Reply:
Thanks @bitwiseman ,

Incidentally, due to the fact that a "chapter" page is a level higher in the URL hierarchy than conventional User Handbook pages (within chapters), I had to implement the following at the top of the Pipeline chapter's `index.adoc` page (which contains the content for the **Pipeline** chapter page itself):

```
ifdef::env-github[:imagesdir: resources]
ifndef::env-github[:imagesdir: ../resources]
````

I've submitted this in a subsequent commit to this PR.

Reply:
@gilesgas  This doesn't seem to be needed. I've reverted that one modification.